apollo-runtime/src/main/java/com/apollographql/apollo/ApolloCall.java

```diff
@@ -128,4 +129,26 @@ public void onCanceledError(@Nonnull ApolloCanceledException e) {
       onFailure(e);
     }
   }
+
+  /**
+   * Represents a status event that corresponds to action that happen to {@link ApolloCall}
+   */
+  enum StatusEvent {
+    /**
+     * {@link ApolloCall} scheduled for execution
```

Comment:
Some comments are past tense some are current. Ie scheduled as opposed to fetches. Maybe standardize.